chapters/ch48.md

```diff
@@ -1,55 +1,57 @@
+# 48. Thêm các ví dụ về phương pháp hoc bắt đầu - kết thúc
 
 > Suppose you want to build a speech recognition system. You might build a system with three components:
 
---> _replace THIS LINE by your translation for the above line_
+Giả thiết bạn muốn xây dựng một hệ thống nhận diện giọng nói. Bạn có thể xây dựng hệ thống này với ba phần:
+
 
 ![img](../imgs/C48_01.png)
 
+
 > The components work as follows:
 
---> _replace THIS LINE by your translation for the above line_
+Các phần này hoạt động như sau:
 
 > 1. Compute features: Extract hand-designed features, such as MFCC (​Mel-frequency cepstrum coefficients) features, ​which try to capture the content of an utterance while disregarding less relevant properties, such as the speaker’s pitch.
 
---> _replace THIS LINE by your translation for the above line_
+1. Phần tính toán các đặc trưng: Trích xuất các đặc trưng thủ công, ví dụ đặc trưng MFCC (hệ số Mel-frequency cepstrum), các đặc trưng này nắm giử nội dung của đoạn phát ngôn trong khi bỏ qua các thuộc tính khác ít liên quan hơn ví dụ như là âm săc của người nói.
 
 > 2. Phoneme recognizer: Some linguists believe that there are basic units of sound called "phonemes." For example, the initial "k" sound in "keep" is the same phoneme as the "c" sound in "cake." This system tries to recognize the phonemes in the audio clip.
 
---> _replace THIS LINE by your translation for the above line_
-
+2. Phần nhận diện các âm vị: Các nhà ngôn ngữ tin rằng trong ngôn ngữ có các đơn vị cơ bản gọi là âm vị. Ví dụ, âm bắt đầu "k" trong từ "keep" thì phát âm giống âm của "c" trong từ "cake". Hệ thống này cố gắng để nhận diện các âm vị trong các audio clip
+ 
 > 3. Final recognizer: Take the sequence of recognized phonemes, and try to string them together into an output transcript.
 
---> _replace THIS LINE by your translation for the above line_
+3. Phần nhận dạng cuối cùng: nhận diện các chuổi âm vị, và cố gắng xâu chuổi chúng với nhau thành một output transcript. 
 
 > In contrast, an end-to-end system might input an audio clip, and try to directly output the transcript:
 
---> _replace THIS LINE by your translation for the above line_
+Ngược lại, một hệ thống hoc bắt đầu - kết thúc đọc đầu vào một audio clip, và cố gắng để cho kêt quả trực tiếp đầu ra là bản dịch:
 
 ![img](../imgs/C48_02.png)
 
 > So far, we have only described machine learning "pipelines" that are completely linear: the output is sequentially passed from one staged to the next. Pipelines can be more complex. For example, here is a simple architecture for an autonomous car:
 
---> _replace THIS LINE by your translation for the above line_
+Từ đầu đến giờ, chúng ta mới mô tả các "chuỗi thuật toán" sử dụng trong học máy hoàn toàn tuyến tính: đầu ra được chuyền đổi theo dạng chuổi từ một trạng thái sang trạng thái tiếp theo. Các "chuỗi thuật toán" này có thể phức tạp hơn. Ví dụ dưới đây là mọt pipline cho một kiến trúc đơn giản của hệ thống lái xe tự động:
```

Comment:
*Từ "end-to-end" và "pipeline" có lẽ phải có một PR riêng thống nhất cách dịch.
* "chuổi" -> "chuỗi"
* Trong câu thứ hai không có chữ "này".
* "hệ thống lái xe tự động" -> "hệ thống xe tự lái".

Reply:
Từ "end-to-end" được dịch là "đầu-cuối", pipeline giữ nguyên.